### PR TITLE
Add reporting documentation

### PR DESCRIPTION
**Pull request by jorblancoa**
Add some documentation about different report types.

### CODE REVIEW

File: source/sonata_report.rst [hunk 1]
```diff
@@ -70,28 +70,180 @@ This field is not implemented.
...
+    * Intrinsic: ``ina``, ``ik``, ``ica``, ``i_pas``, ``i_cap``, ``icsa_csa``, ``ihcn_Ih``, ``i_membrane``
+    * Synaptic: ``ProbAMPANMDA_EMS``, ``ProbGABAA_EMS``
+    * Stimulus: ``IClamp``, ``SEClamp``
+
+For summation reports, you need to specify ``"type": "summation"`` in the configuration::
+
+    "reports": {
+      "cell_imembrane": {
+         "cells": "Column",
+         "variable_name": "i_membrane,IClamp",  # Multiple variables to sum
```
Comment (WeinaJi) [inline source/sonata_report.rst:123]: Can we add an example of synaptic current? To illustrate the naming convention.

File: source/sonata_report.rst [hunk 2]
```diff
@@ -70,28 +70,180 @@ This field is not implemented.
...
+
+* Electrode positions must be defined before simulation
+* Uses same HDF5 structure as compartment reports
+* element_ids correspond to electrode positions
+* Commonly used for analyzing population-level activity
+
+For more detailed information about LFP reports, please refer to the `Online LFP Calculation Documentation <https://github.com/BlueBrain/neurodamus/blob/main/docs/online-lfp.rst#online-lfp-calculation-documentation>`_.
 
 Bloodflow report
 ^^^^^^^^^^^^^^^^
```
Comment (WeinaJi) [inline source/sonata_report.rst:249]: We support bloodflow report? What is the report type ?

**Review by WeinaJi — COMMENTED** [review]
Thanks for adding more details.

File: source/sonata_report.rst (hunk 2 above)
Comment (jorblancoa) [reply source/sonata_report.rst:259]: This was not added by me. Those are not done in the conventional way. 
tbh I dont know how is this generated
@cattabiani probably knows?

Comment (cattabiani) [reply source/sonata_report.rst:259]: I did not add this. Maybe Dan? I cannot @

Comment (cattabiani) [reply source/sonata_report.rst:259]: maybe it comes from vascCouplingB?

Comment (WeinaJi) [reply source/sonata_report.rst:259]: This type of report is created by the bloodflow solver. 
https://github.com/BlueBrain/AstroVascPy/blob/d40a16b78bf5c4b1f02b8d8f6f8cb30b316b9dd9/astrovascpy/report_writer.py#L19
It is clear now, we keep it like this.

**Review by WeinaJi — APPROVED** [review]

**Pull request merged**